packages/mariadb-server-10.1-mroonga/Rakefile

```
@@ -0,0 +1,27 @@
+require_relative "../mroonga-package-task"
+
+class MariaDBServer101MroongaPackageTask < MroongaPackageTask
+  def initialize
+    super("mariadb-server-10.1", "mariadb-10.1-mroonga")
+  end
+
+  def apt_targets_default
+    [
+      "debian-stretch",
+    ]
+  end
```

```suggestion
  def enable_apt?
    false
  end
```

Stretch reached EOL.